widgets/Ariba-batchsource.json

```diff
@@ -47,15 +47,15 @@
           "label": "Realm",
           "name": "realm",
           "widget-attributes": {
-            "placeholder": "Specify the realm name for the customer. E.g. : S4SVAll"
+            "placeholder": "Target Realm name from which data is to be extracted. E.g. : S4SVAll"
```

Comment:
Remove "Target" and spell out for example. "Realm name from which the data is to be extracted, for example S4VALL."